### PR TITLE
852 analytical frequency curve should plot on normal probability axis

### PR DESCRIPTION
**Pull request by Brennan1994**

### CODE REVIEW

**Comment by Brennan1994** [comment]
This does not fully address the issue of confidence limits yet. It applies them in one location, and adds the model logic to compute them.

File: HEC.FDA.ModelTest/unittests/extensions/ContinuousDistributionExtensionsShould.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+        [Fact]
+        //https://www.hec.usace.army.mil/confluence/sspdocs/ssptutorialsguides/expected-moments-algorithm-study-guide/user-defined-functions-for-lpiii-and-gev-frequency-curves#UserDefinedFunctionsforLPIIIandGEVFrequencyCurves-Exercise
```
Comment (rnugent3) [inline HEC.FDA.ModelTest/unittests/extensions/ContinuousDistributionExtensionsShould.cs:20]: cool!

**Comment by rnugent3** [comment]
I ran into an uncaught exception 

image

**Comment by rnugent3** [comment]
The exception happens when I attempt to view a graphical frequency function.

**Review by rnugent3 — CHANGES_REQUESTED** [review]
Please troubleshoot the overflow exception.

**Comment by Brennan1994** [comment]
The latest update updates the tracker to use the probability rather than the Z score, so you can check exact probabilities anywhere on the lines. 

I was unable to reproduce your error. Please send me your dataset if that's still an issue.

**Review by rnugent3 — APPROVED** [review]
terrific

**Pull request merged**